models/locale.php

```diff
@@ -19,19 +19,22 @@
     $locale = $_GET['locale'];
 }
 
-// get lang files status from langchecker
-$lang_files = Json::fetch(LANG_CHECKER . "?locale={$locale}&json");
+// Get lang files status from langchecker
+$json_data = new Json();
+$json_data->setURI(LANG_CHECKER . "?locale={$locale}&json");
+$lang_files = $json_data->fetchContent();
```

Comment:
Tried but it didn't work. Let me try again, maybe I just screwed up something silly.